client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1992,8 +1992,5 @@
   "warning.caps-lock": "Warning: Caps lock is on",
   "warning.check-before-vaccinating": "Vaccination records from other stores may not be up to date. Ensure you have synced recently, and always confirm against the patient's physical vaccination card first.",
   "warning.field-not-parsed": "{{field}} not parsed",
-  "warning.nothing-to-supply": "Nothing left to supply!",
-  "description": {
```

Comment:
this was definitely not used somehwere in a report right?

Reply:
Pretty confident it's not, on develop I just changed it to `description.suggested-quantity` as I think that was the intention, thought i'd fixed this branch but clearly hadn't...